tests/test_verifier.py

```diff
@@ -77,3 +77,9 @@ def test_untrusted_cert_url_domain(self):
             'SigningCertURL': 'https://www.example.com/',
         })
         self.assertEqual(verifier._get_cert_url(), None)
+
+    def test_get_bytes_to_sign(self):
```

Comment:
Did you verify this test fails on python2 before your fix? Otherwise this PR looks good, thanks for contributing 👍

Reply:
Yep the test failed on Python 2 before the other changes.